english/LifePuzzle/LifePuzzle.md

```diff
@@ -5,4 +5,11 @@ and your mass is proportionally reduced
 so as to maintain your original density. 
 You are then thrown into an empty glass blender. 
 The blades will start moving in 60 seconds. 
-What do you do?
\ No newline at end of file
+What do you do?
+
+Drink some [bear](bear/bear.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.